<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Malware_Dialog</class>
 <widget class="QDialog" name="Malware_Dialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>626</width>
    <height>454</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>626</width>
    <height>454</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>626</width>
    <height>454</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Upload Malware</string>
  </property>
  <property name="statusTip">
   <string>Upload Exploit</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog { background-color: qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, stop:0 rgba(17, 5, 0, 255), stop:0.081448 rgba(66, 46, 24, 255), stop:0.104072 rgba(30, 26, 73, 255), stop:0.41629 rgba(7, 4, 37, 255), stop:0.714932 rgba(126, 14, 81, 255), stop:0.751131 rgba(29, 13, 13, 255), stop:0.886878 rgba(70, 70, 51, 255)); }</string>
  </property>
  <property name="windowFilePath">
   <string/>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QDialogButtonBox" name="UW_buttonBox">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>400</y>
     <width>191</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>601</width>
     <height>101</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>ubuntu-light</family>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="statusTip">
    <string>Platform</string>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QGroupBox { font-family: ubuntu-light; border: 2px solid gray;  border-radius: 5px; margin-top: 1ex; background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(228, 251, 79, 206), stop:0.4 rgba(244, 252, 255, 241), stop:0.425 rgba(244, 252, 255, 241), stop:0.44 rgba(244, 252, 255, 241), stop:1 rgba(244, 252, 255, 241)); }</string>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhNone</set>
   </property>
   <property name="title">
    <string>Platform</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>60</y>
      <width>259</width>
      <height>25</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QRadioButton" name="x86_Radio">
       <property name="font">
        <font>
         <weight>75</weight>
         <italic>true</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>&amp;x86</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="x64_Radio">
       <property name="font">
        <font>
         <weight>75</weight>
         <italic>true</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>x6&amp;4</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="x86_64_Radio">
       <property name="font">
        <font>
         <weight>75</weight>
         <italic>true</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>x86_&amp;64</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="ALL_Radio">
       <property name="font">
        <font>
         <weight>75</weight>
         <italic>true</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>A&amp;LL</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>201</width>
      <height>34</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="Platform_label">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Target platform:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="Platform_LineEdit">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>20</y>
      <width>201</width>
      <height>34</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="Version_Label">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Target Version</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="Version_Edit">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>601</width>
     <height>281</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>ubuntu-light</family>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
     <underline>false</underline>
     <strikeout>false</strikeout>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QGroupBox { font-family: ubuntu-light; border: 2px solid gray;  border-radius: 5px; margin-top: 1ex; background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(228, 251, 79, 206), stop:0.4 rgba(244, 252, 255, 241), stop:0.425 rgba(244, 252, 255, 241), stop:0.44 rgba(244, 252, 255, 241), stop:1 rgba(244, 252, 255, 241)); }</string>
   </property>
   <property name="title">
    <string>Information</string>
   </property>
   <widget class="QComboBox" name="Type_Combo">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>40</y>
      <width>75</width>
      <height>33</height>
     </rect>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::CustomContextMenu</enum>
    </property>
    <property name="currentText">
     <string/>
    </property>
    <property name="maxVisibleItems">
     <number>3</number>
    </property>
   </widget>
   <widget class="QComboBox" name="Ratings_Combo">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>110</y>
      <width>75</width>
      <height>33</height>
     </rect>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::CustomContextMenu</enum>
    </property>
    <property name="currentText">
     <string/>
    </property>
    <property name="maxVisibleItems">
     <number>3</number>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>201</width>
      <height>34</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="Malware_Title_Label">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Malware title: </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="Malware_Title_Edit">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_4">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>70</y>
      <width>421</width>
      <height>36</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QLabel" name="Malware_Module__Label">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Malware Module:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="Malware_Module_Edit">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Malware_Module_Button">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="mouseTracking">
        <bool>true</bool>
       </property>
       <property name="tabletTracking">
        <bool>true</bool>
       </property>
       <property name="acceptDrops">
        <bool>true</bool>
       </property>
       <property name="statusTip">
        <string>Exploit Module</string>
       </property>
       <property name="text">
        <string>Module</string>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_7">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>110</y>
      <width>421</width>
      <height>36</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_8">
     <item>
      <widget class="QLabel" name="Malware_Options_Label">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Malware Options:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="Malware_Opt_Edit">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Malware_JSON_Push">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="mouseTracking">
        <bool>true</bool>
       </property>
       <property name="tabletTracking">
        <bool>true</bool>
       </property>
       <property name="acceptDrops">
        <bool>true</bool>
       </property>
       <property name="statusTip">
        <string>Exploit Module</string>
       </property>
       <property name="text">
        <string>JSON</string>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_5">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>30</y>
      <width>231</width>
      <height>34</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QLabel" name="Malware_Author_Label">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Author Email:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="Malware_Author_Edit">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>150</y>
      <width>491</width>
      <height>118</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="Malware_Description_Label">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Malware Description</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextEdit" name="Malware_Description_TextEdit"/>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>20</y>
      <width>54</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Type</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>90</y>
      <width>54</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Ratings</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>UW_buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Malware_Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>UW_buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Malware_Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
